converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/ImagingStudy/ImagingStudyPropertySynchronizer.cs

```diff
@@ -100,5 +102,27 @@ private void AddAccessionNumber(ImagingStudy imagingStudy, DicomDataset dataset)
                 }
             }
         }
+
+        private void AddNumberOfSeries(ImagingStudy imagingStudy, DicomDataset dataset)
+        {
+            if (dataset.TryGetSingleValue(DicomTag.NumberOfStudyRelatedSeries, out int numberOfSeries))
+            {
+                if (imagingStudy.NumberOfSeries == null)
```

Comment:
so we would not update it? why? suppose we added more instances and we got a new instance with updated series or instance why would we not updaye it?

Reply:
Changed to always update so we assume latest uploaded is source of truth.